sequins.go

```diff
@@ -100,6 +107,13 @@ func (s *sequins) init() error {
 	}
 	s.workQueue = workqueue.NewWorkQueue(maxLoads)
 
+	// Set up token bucket for download bandwidth throttling
+	maxDownloadBandwidth := s.config.MaxDownloadBandwidthMBPerSecond * 1024 * 1024
```

Comment:
Can we just not introduce this artificial limit and just pass nil to bucket?

In the downloader, if the bucket is nil, we can skip the reader wrap operation.

Reply:
Okay. 

In my opinion, we should throttling by default. The downloading part has lower priority. It should not compete with the serving part on bandwidth and CPU.